drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -3377,8 +3359,7 @@ static int mmcsd_probe(FAR struct mmcsd_state_s *priv)
             {
               /* Yes...  */
 
-              finfo("Capacity: %lu Kbytes\n",
-                    (unsigned long)(priv->capacity / 1024));
+              finfo("Capacity: %lu blocks\n",  (unsigned long)priv->nblocks);
```

Comment:
I don't think this is a good change! You are replacing human readable Kbytes by number of blocks. It makes user life more complicated. Please keep the Capacity in Kbytes/Mbytes.

Reply:
Ok, the capacity bring back again.